src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -119,16 +129,15 @@ function make(config: Config): EthereumBlockchainService {
   let provider
   if (url) {
     logger.imp(`Connecting ethereum provider to url: ${url}`)
-    provider = new ethers.providers.StaticJsonRpcProvider(url)
+    provider = new ethers.JsonRpcProvider(url)
```

Comment:
the reason we were using `StaticJsonRpcProvider` was to avoid excessive numbers of calls to `eth_chainid` as per this bug:  https://github.com/ethers-io/ethers.js/issues/901

If the static version doesn't exist anymore, we should make sure to double-check that this change doesn't result in a ton of eth_chainid calls again.

Reply:
Will have to check. The static provider doesn't seem to exist anymore.

Reply:
@dbcfd There is an option that emulates `StaticJsonRpcProvider`: https://docs.ethers.org/v6/api/providers/jsonrpc/#JsonRpcApiProviderOptions

Reply:
@ukstv Looking at the docs, that looks to default to null, which would prevent the eth_chainid calls, so I don't think any change is needed here?

Reply:
@dbcfd Last time I read ethers 6 code `null` there meant that it would try to get chain id on every call. If the parameter is absent (default: `null`) means we would struggle with excessive chainId calls.